lzero/policy/random_policy.py

```diff
@@ -165,26 +206,47 @@ def _forward_collect(
 
             for i, env_id in enumerate(ready_env_id):
                 distributions, value = roots_visit_count_distributions[i], roots_values[i]
+
+                if self._cfg.type in ['sampled_efficientzero']:
+                    try:
+                        root_sampled_actions = np.array([action.value for action in roots_sampled_actions[i]])
+                    except Exception:
```

Comment:
why use this implementation, can we avoid `try-except` in algorithm codes as possible as we can

Reply:
In ctree, the method roots.get_sampled_actions() returns a list object. However, in ptree, the same method roots.get_sampled_actions() returns an Action object. Now we have polished this part.